app/Models/Product.php

```diff
@@ -49,4 +55,16 @@ public function orderDetails()
     {
         return $this->hasMany(OrderDetail::class);
     }
+
+    public static function filterProduct()
+    {
+        $products = Product::query();
+
+        $pipe = Pipeline::send($products)->through([
+            \App\QueryFilters\Active::class,
+            \App\QueryFilters\Sort::class,
```

Comment:
m `use` các filter cần thiết ở trên cùng nhé
```suggestion
            Active::class,
            Sort::class,
```